ocfweb/docs/docs/staff/policies/keycard.md

```diff
@@ -0,0 +1,54 @@
+[[!meta title="Keycard policy"]]
+
+<!-- This policy was promulgated by the GMs on 8/18/2017. -->
+
+
+## Keycard Access
+
+Keycard access is given much like root, but requires approval from both GMs and
+SMs.
+
+To be eligible for Keycard access, you must fulfill the following criteria:
+
+ - A Site Manager/General Manager must be able to recognize you in person
+ - You must have been a staffer for at least a year already, but this may be
+   waived at the discretion of the GM/SM
+ - You must demonstrate a concrete, actual need for keycard privileges and
+   concrete examples of helpful activity in the OCF
+
+Keycard Access, once given, can be revoked at the discretion of SMs or GMs.
```

Comment:
`Access` -> `access`? why are these things being arbitrarily capitalized in only some places? lol

Reply:
Ask Sahil...